unity/Assets/Scripts/UI/UIWindowSelectionListTraits.cs

```diff
@@ -291,14 +291,14 @@ public void AddItem(GenericData component, Color color)
             items[items.Count - 1].SetColor(color);
         }
 
-        public void AddNewComponentItem(string type)
+        public void AddNewComponentItem(StringKey type)
         {
             Dictionary<string, IEnumerable<string>> traits = new Dictionary<string, IEnumerable<string>>();
 
-            traits.Add(new StringKey("val", "TYPE").Translate(), new string[] { type });
```

Comment:
I would rather just fix this here with:
> traits.Add(new StringKey("val", "TYPE").Translate(), new string[] { type.ToUpper()).Translate() });

We are doing this anyway on line 301 and it keeps all of the interface text here rather than in the calling functions.  The type is "Tile" (for example), not a string key.  Unless I am missing something this one line change (and the "Free->FREE" fix) should address the issue?

I would also like to target this at master for a bug fix release.